website/docs/getting-started/setup/installation-guides/kubernetes/publish-appsmith-online.mdx

```diff
@@ -8,125 +8,89 @@ import TabItem from '@theme/TabItem';
 The page provides steps to publish Appsmith Kubernetes installation to the internet.
```

Comment:
i think we should change the title to "Publish K8s to Internet" or "Expose K8s to Internet" We don't need to specify Appsmith installation and all

Reply:
Expose k8s to Internet